src/bin/stratis-utils.rs

```diff
@@ -166,6 +171,64 @@ fn parse_args() -> Result<(), Box<dyn Error>> {
                 .map(|bs| bs.map(PathBuf::from).collect::<Vec<_>>())
                 .unwrap_or_default(),
         )?;
+    } else if argv1.ends_with("stratis-clevis-setup-generator")
+        || argv1.ends_with("stratis-setup-generator")
+    {
+        #[cfg(feature = "systemd_compat")]
+        if argv1.ends_with("stratis-clevis-setup-generator") {
+            let parser = App::new("stratis-clevis-setup-generator")
+                .arg(
+                    Arg::with_name("normal_priority_dir")
+                        .required(true)
+                        .help("Directory in which to write a unit file for normal priority"),
+                )
+                .arg(
+                    Arg::with_name("early_priority_dir")
+                        .required(true)
+                        .help("Directory in which to write a unit file for early priority"),
+                )
+                .arg(
+                    Arg::with_name("late_priority_dir")
+                        .required(true)
+                        .help("Directory in which to write a unit file for late priority"),
+                );
+            let matches = parser.get_matches_from(&args);
+            stratis_clevis_setup_generator::generator(
```

Comment:
I can't make sense of three required arguments, only one of which is used.

Reply:
This is the systemd generator interface. It is mandatory to accept three positional arguments.